external/storm-druid/src/main/java/org/apache/storm/druid/bolt/DruidBeamBolt.java

```diff
@@ -76,30 +77,33 @@ public void prepare(Map stormConf, TopologyContext context, OutputCollector coll
 
     @Override
     public void execute(final Tuple tuple) {
-      Future future = tranquilizer.send((druidEventMapper.getEvent(tuple)));
-        LOG.debug("Sent tuple : [{}]" , tuple);
+        if (TupleUtils.isTick(tuple)) {
```

Comment:
@satishd 
Make sense.
About class name I don't see good name for that. I thought `TickTupleAwareBaseRichBolt` inspired of Spring but the naming `~Aware` is actually used for interfaces. 
If we would want to make it public API (instead of using internally), it needs to have better class name.

@dossett 
`execute()` also doesn't have `throws` definitions. If we follow @satishd  `execute()` implementation, `execute()` doesn't care about exceptions and it's up to  subclasses.
So handling exceptions for two separated methods (`onTickTuple()` and `process()`) could happen, but it doesn't hurt. It just makes some duplicate codes.

Do you have other considerations or point of view?